src/index.html

```diff
@@ -7,21 +7,425 @@
+
+          <div class="card__review-container">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <div class="card__review-rate">Reviews: 5</div>
+          </div>
+
+          <div class="card__price-container">
+            <div class="card__price-label">Price:</div>
+            <div class="card__price">$2,199</div>
+          </div>
+
+          <a
+            class="card__button"
+            href="#"
+          >
+            Buy
+          </a>
+        </div>
+
+        <div class="card cards__card">
+          <img
+            class="card__picture"
+            src="./images/imac.jpeg"
+            alt="Apple-A1419-iMac"
+          />
+
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <div class="card__code">Product code: 195434</div>
+
+          <div class="card__review-container">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <div class="card__review-rate">Reviews: 5</div>
+          </div>
+
+          <div class="card__price-container">
+            <div class="card__price-label">Price:</div>
+            <div class="card__price">$2,199</div>
+          </div>
+
+          <a
+            class="card__button"
+            href="#"
+          >
+            Buy
+          </a>
+        </div>
+
+        <div class="card cards__card">
+          <img
+            class="card__picture"
+            src="./images/imac.jpeg"
+            alt="Apple-A1419-iMac"
+          />
+
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <div class="card__code">Product code: 195434</div>
+
+          <div class="card__review-container">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <div class="card__review-rate">Reviews: 5</div>
+          </div>
+
+          <div class="card__price-container">
+            <div class="card__price-label">Price:</div>
+            <div class="card__price">$2,199</div>
+          </div>
+
+          <a
+            class="card__button"
+            href="#"
+          >
+            Buy
+          </a>
+        </div>
```

Comment:
The product card elements are repeated multiple times with the same content. If this is intentional for demonstration purposes, it's fine. However, if this is part of a dynamic web application, consider generating these cards dynamically using a template engine or JavaScript to avoid code duplication and improve maintainability.